<?xml version="1.0"?>
<launch>
    <!-- Высота взлета над землей в метрах -->
    <arg name="altitude" default="1.5" />
    <!-- Использовать визуальную одометрию или нет -->
    <arg name="use_vision_odometry" default="0" />
    <!-- Взлетать при получении сообщений в топик cmd_vel -->
    <arg name="takeoff_on_msg" default="1" />
    <!-- Тип автопилота -->
    <arg name="ap_type" default="0" />
    
    <node name="$(anon cmd_vel_remapper)" pkg="autotakeoff" type="cmd_vel_remapper_node.py" output="log">
        <param name="altitude" value="$(arg altitude)" />
        <param name="ap_type" value="$(arg ap_type)" type="bool"/>
        <param name="use_vision_odometry" value="$(arg use_vision_odometry)" type="bool"/>
        <param name="takeoff_on_msg" value="$(arg takeoff_on_msg)" type="bool"/>
    </node>
</launch>